src/main.js

```diff
@@ -58,6 +68,22 @@ function displayCurrentCover() {
   `
 }
 
+function displayMiniCovers() {
+  var miniCoversHTML = "";
+  savedCovers.forEach(function(cover) {
+    miniCoversHTML += `
+      <img class="main-cover" src=${cover.cover}>
+         <h2 class="cover-title">${cover.title}</h2>
+         <h3 class="tagline">A tale of <span class="tagline-1">${cover.tagline1}</span> and <span class="tagline-2">${cover.tagline2}</span></h3>
+         <img class="price-tag" src="./assets/price.png">
+         <img class="overlay" src="./assets/overlay.png">
+    `
+  })
+  savedCoversSection.classList.add("mini-cover");
+  savedCoversSection.innerHTML = miniCoversHTML;
+}
+
+
```

Comment:
I think we're on the right path with this function - I think with a little help we should be able to get the covers to display correctly!